go.mod

```diff
@@ -6,20 +6,20 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.4.15-0.20200113171025-3fe6c5262873
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/containerd v1.3.4
+	github.com/containerd/containerd v1.3.6
 	github.com/containerd/ttrpc v1.0.1
-	github.com/containernetworking/cni v0.7.2-0.20200304161608-4fae32b84921
+	github.com/containernetworking/cni v0.8.0
 	github.com/containernetworking/plugins v0.8.6
-	github.com/containers/buildah v1.14.9
-	github.com/containers/common v0.9.1
-	github.com/containers/conmon v2.0.16+incompatible
-	github.com/containers/image/v5 v5.4.4
-	github.com/containers/libpod v1.9.2
+	github.com/containers/buildah v1.15.0
+	github.com/containers/common v0.14.0
```

Comment:
is there a reason to go to 0.14 and not 0.15?

Reply:
I think the release happened after this PR, I can update it when the times comes :)